eng/scripts/dependencies/generate-deps.ps1

```diff
@@ -0,0 +1,36 @@
+param(
+  [string]$PackagesPath,
+  [string]$DepsOutputFile,
+  [string]$ProjectRefPath
+)
+
+#-Packages (dir -include "$(System.ArtifactsDirectory)/BuildArtifacts/Azure.*.nupkg" -exclude *symbols.nupkg)
+#-DepsOutputFile "$(Build.ArtifactStagingDirectory)/reports/azure-sdk.deps.json"
+
+mkdir $ProjectRefPath -force | out-null
+$projRefsPath = Join-Path $ProjectRefPath "azure-sdk-projectreferences.props"
+
+$projectRefsContents = @"
```

Comment:
So we don't already have a project that includes everything anymore, and must create one?

Reply:
Correct we don't have one that contains everything and even the one we did have was error prone.